pkg/agent/plugin/workloadattestor/unix/unix.go

```diff
@@ -217,6 +231,20 @@ func (p *Plugin) getGroupName(gid string) (string, bool) {
 	return g.Name, true
 }
 
+func (p *Plugin) getSupplementaryGroupsIDs(proc processInfo) ([]string, error) {
+	groups, err := proc.Groups()
```

Comment:
If `proc.Groups()` is not currently working for mac, perhaps we could detect the platform (using `runtime.GOOS`) and avoid the function call?

In that way, we could display a more friendly error message instead of the current "index out of bound".
Maybe something explaining that those selectors are not currently available for Mac OS.

Reply:
💯 Avoiding the call on mac sounds like a good approach to me.

Reply:
and can you add a comment into code, because why of that and maybe a link to that issue?